server-timing/class-perflab-server-timing.php

```diff
@@ -195,6 +195,10 @@ static function( $value ) {
 	 * @return bool True if an output buffer should be used, false otherwise.
 	 */
 	public function use_output_buffer() {
+		$slug    = 'output_buffering';
+		$options = (array) get_option( PERFLAB_SERVER_TIMING_SETTING, array() );
+		$enabled = ! empty( $options[ $slug ] );
```

Comment:
Nit-pick: No need for the `$slug` variable as it's only used once.
```suggestion
		$options = (array) get_option( PERFLAB_SERVER_TIMING_SETTING, array() );
		$enabled = ! empty( $options['output_buffering'] );
```